package.json

```diff
@@ -39,31 +39,27 @@
     }
   },
   "devDependencies": {
+    "@aws-cdk/aws-apigatewayv2-alpha": "2.114.1-alpha.0",
+    "@aws-cdk/aws-apigatewayv2-integrations-alpha": "2.114.1-alpha.0",
     "@qiwi/semantic-release-gh-pages-plugin": "^5.2.3",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
     "@types/node": "^18.7.23",
-    "jsii": "5.7.4",
+    "aws-cdk-lib": "2.130.0",
+    "constructs": "10.3.0",
     "jsii-docgen": "10.6.1",
     "jsii-pacmak": "1.106.0",
+    "jsii": "5.7.4",
     "nodemon": "^2.0.20",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.7.1",
-    "semantic-release": "^19.0.5",
-    "aws-cdk-lib": "2.130.0",
-    "constructs": "10.3.0",
-    "@aws-cdk/aws-apigatewayv2-alpha": "2.114.1-alpha.0"
-  },
-  "dependencies": {
-    "@aws-cdk/aws-apigatewayv2-integrations-alpha": "2.114.1-alpha.0",
-    "@aws-cdk/aws-apigatewayv2-alpha": "2.114.1-alpha.0",
-    "aws-cdk-lib": "2.130.0",
-    "constructs": "10.3.0"
+    "semantic-release": "^19.0.5"
   },
   "peerDependencies": {
-    "aws-cdk-lib": "2.130.0",
-    "constructs": "10.3.0",
-    "@aws-cdk/aws-apigatewayv2-alpha": "2.114.1-alpha.0"
+    "@aws-cdk/aws-apigatewayv2-alpha": "^2.114.1-alpha.0",
```

Comment:
we want to pin to a specific version of the `alpha` dependencies, right?

Reply:
Pinned in dd7281dcaea058fcee392ce2b2fdef98c2684290